src/views/Layout.jsx

```diff
@@ -1,52 +1,52 @@
-import { Outlet, NavLink } from 'react-router-dom';
+import { Outlet } from 'react-router-dom';
 
 import './Layout.css';
 import { auth } from '../api/config.js';
-import { SignInButton, SignOutButton, useAuth } from '../api/useAuth.jsx';
+import { SignInButton, useAuth } from '../api/useAuth.jsx';
+import { NavBar } from '../components/NavBar.jsx';
+import { SignOutButton } from '../api/useAuth.jsx';
 
 export function Layout({ lists, listPath }) {
 	const { user } = useAuth();
 
+	const handleClickSignIn = () => {
+		SignInButton();
+	};
+
 	return (
-		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+		<div className="h-dvh flex flex-col text-poppins">
+			<NavBar user={user} lists={lists} listPath={listPath} />
+			<header className="Layout-header">
+				{/* Example how to use the font families and the colors with tailwindCSS */}
+				{/* example how to use awesome font. To style the icon, wrap it around a span and use tailwindCSS */}
+				<span className="text-alertRed">
+					<i className="fa-solid fa-trash"></i>
+					<i className="fa-solid fa-share-nodes"></i>
+					<i className="fa-solid fa-plus"></i>
+					<i className="fa-solid fa-xmark"></i>
+				</span>
 
-					{!!user ? (
-						<>
-							<p>{`Welcome ${user?.displayName}`}</p>
-							<SignOutButton />
-						</>
-					) : (
-						<>
-							<h3>Log in to begin using the shopping list app</h3>
-							<SignInButton />
-						</>
-					)}
-				</header>
-				<main className="Layout-main">{!!user ? <Outlet /> : null}</main>
-				{!!user && (
-					<nav className="Nav">
-						<div className="Nav-container">
-							<NavLink to="/" className="Nav-link">
-								Home
-							</NavLink>
-							{!!listPath && !!lists[0] && (
-								<>
-									<NavLink to={`/list/${listPath}`} className="Nav-link">
-										List
-									</NavLink>
-									<NavLink to="/manage-list" className="Nav-link">
-										Manage List
-									</NavLink>
-								</>
-							)}
-							<SignOutButton />
-						</div>
-					</nav>
+				{!!user ? (
+					<>
+						<p>{`Welcome ${user?.displayName}`}</p>
+						{/* <button onClick={SignOutButton()}>sign out</button> */}
+						{/* <SignOutButton /> */}
+					</>
+				) : (
+					<>
+						<h3>Log in to begin using the shopping list app</h3>
+						<button
+							className="font-poppins text-lg"
+							type="button"
+							onClick={handleClickSignIn}
+						>
+							Sign In
+						</button>
+						{/* <SignInButton /> */}
+					</>
 				)}
-			</div>
-		</>
+			</header>
+			<main className="Layout-main">{!!user ? <Outlet /> : null}</main>
```

Comment:
I think this is a good place to set a max-withd, so we don't have to do so on each view

Reply:
The layout page needs some work on it, I will create an issue for it.